files/lib/data/conversation/ConversationAction.class.php

```diff
@@ -831,6 +836,100 @@ public function getMixedConversationList()
         ];
     }
 
+    /**
+     * @since 5.5
+     */
+    public function validateGetConversations(): void
+    {
+    }
```

Comment:
I wonder if these validateX methods would need to check the MODULE and possibly the group permissions. One might have had access to conversations in the past, but then later lost the permissions to access them.